src/Search/Search.Tests/Tests/CustomAnalyzerTests.cs

```diff
@@ -284,19 +284,19 @@ public void CanCreateAllAnalysisComponents()
                         maxTokenLength: 100, 
                         isSearchTokenizer: true, 
                         language: MicrosoftTokenizerLanguage.Thai),
-                    new PathHierarchyTokenizer(
```

Comment:
>PathHierarchyTokenizer [](start = 24, length = 22)

Seems PathHierarchyTokenzier will no longer be tested. Shouldn't these be left for regression purpose, until the entire class has been deleted.